install.sh

```diff
@@ -39,8 +39,7 @@ INSTALL_DIR="/usr/share/doc/hackingtool"
 BIN_DIR="/usr/bin/"
 if [ $choice == 1 ]; then 
 	echo "[*] Checking Internet Connection .."
-	wget -q --tries=10 --timeout=20 --spider https://google.com
-	if [[ $? -eq 0 ]]; then
+  if ping -c 1 -w 20 8.8.8.8 > /dev/null;then
```

Comment:
In shell, you need to close each `if` with a corresponding `fi`.  Also, indentation is not required but it is recommended to help readers understand the intent.

Reply:
Yes, it's my feault.

Reply:
Yes, it's my fault.

Reply:
If you add an `if`, you must add a corresponding `fi`.

Reply:
No, I'm just modified the condition of  `if`.